src/views/Recents/services/fetchRecents.test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it, vi, beforeEach, Mock } from 'vitest';
+import { fetchRecents } from './fetchRecents';
+import { SdkFactory } from 'app/core/factory/sdk';
+
+vi.mock('app/core/factory/sdk', () => ({
+  SdkFactory: {
+    getNewApiInstance: vi.fn(),
+  },
+}));
+
+describe('fetchRecents', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('should call getRecentFilesV2 with the correct limit', async () => {
```

Comment:
nit: better use a non-technical description for tests. Imagine that the function is renamed and now it is called getRecentFiles instead, you will update the tests because it will fail (or not) but the test description will cause confusion.